docs/issuing-assets/how-to-issue-an-asset.mdx

```diff
@@ -102,30 +295,185 @@ const transaction = new StellarSdk.TransactionBuilder(...)
   }))
 ```
 
+```java
+Transaction transaction = new Transaction.Builder(...)
+  .addOperation(new PaymentOperation.Builder(
+    distributorKeypair.getAccountId(),
+    astroDollar,
+    "1000"
+  )
+  .setSourceAccount(
+    issuerKeypair.getAccountId()
+  )
+```
+
+```go
+transaction, _ := txnbuild.NewTransaction(
+  txnbuild.TransactionParams{
+    SourceAccount:        &issuerAccount,
+    IncrementSequenceNum: true,
+    BaseFee:              txnbuild.MinBaseFee,
+    Operations: []txnbuild.Operation{
+      &txnbuild.Payment{
+        Destination: distributorKeypair.Address(),
+        Amount:      "1000",
+        Asset:       astroDollar,
+      },
+    },
+    Timebounds: txnbuild.NewInfiniteTimeout(),
+  },
+)
+```
+
+```python
```

Comment:
It would probably be a good idea in all the code examples in this step to add a comment in the snippet, something like...

```python
transaction = (
    # We're using TransactionBuilder(...) as a short-hand here to show that these operations can be "chained" together
    TransactionBuilder(...)
    .append_payment_op(
```

Reply:
Also, just realizing now that your python is only indented by 2 spaces. We want that to be 4.

Reply:
That's a really good point you bring up. Our [JS prettier config](https://github.com/stellar/product-conventions/blob/master/packages/prettier-config/prettier.config.js) uses 2 spaces, but for Python we don't really have anything stating that 4 spaces are required. Again, just falling back to the norms and conventions of the broader Python community there.

We've got on the (long) roadmap that we need to come up with some of those codified, stated, enforced choices. In the meantime, I don't have a strong enough opinion to let python tab-width block a PR from being merged lol.